aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,14 +1,34 @@
 ---
 remote_endpoint: ~
-name: "v1.8"
+name: "v1.9"
 proposals:
-  - name: upgrade_framework
+  - name: step_1_upgrade_framework
     metadata:
-      title: "Multi-step proposal to upgrade mainnet framework to v1.8"
-      description: "This includes changes in https://github.com/aptos-labs/aptos-core/commits/aptos-release-v1.8"
+      title: "Multi-step proposal to upgrade mainnet framework to v1.9"
+      description: "This includes changes in https://github.com/aptos-labs/aptos-core/commits/aptos-release-v1.9"
     execution_mode: MultiStep
     update_sequence:
       - DefaultGas
       - Framework:
           bytecode_version: 6
           git_hash: ~
+  - name: step_2_commission_change_delegation_pool
+    metadata:
+      title: "AIP-50: Change commission rates in delegation pools"
+      description: "AIP-50: This allows delegation pool owners to change operator commission rates after delegation pool creation."
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/249"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
+            - commission_change_delegation_pool
+  - name: step_3_enable_automatic_sponsored_account_creation
+    metadata:
+      title: "AIP-52: Automated Account Creation for Sponsored Transactions"
+      description: "AIP-52: Automated Account Creation for Sponsored Transactions."
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/258"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
+            - sponsored_automatic_account_creation
```

Comment:
@junkil-park can you add AIP-57 to the list (not sure about the `block_output_limit: Some(5 * 1024 * 1024)` part)

```

  metadata: 
    title: "AIP-57: Block Output Size Limit and Conflict-Aware Block Gas Limit"
    description: "AIP-57: Block Output Size Limit and Conflict-Aware Block Gas Limit"
    discussion_url: "https://github.com/aptos-foundation/AIPs/issues/285"
  update_sequence:
      - Execution:
          V4:
            transaction_shuffler_type:
              sender_aware_v2: 32
            block_gas_limit_type: 
              complex_limit_v1:
                effective_block_gas_limit: 40000
                execution_gas_effective_multiplier: 1
                io_gas_effective_multiplier: 1
                conflict_penalty_window: 6
                use_granular_resource_group_conflicts: false
                use_module_publishing_block_conflict: false
                block_output_limit: Some(5 * 1024 * 1024)
                include_user_txn_size_in_block_output: true
                add_block_limit_outcome_onchain: false
            },
            transaction_deduper_type: txn_hash_and_authenticator_v1
```

Reply:
@igor-aptos, could you check the syntax locally? It would need the step name like "- name: step_4_block_output_size_and_gas_limit", and it would need to fix the unnecessary closing curly brace "},".

Reply:
if you want to land this, I can just send a separate PR, or I'll share it later. 

every typo I have takes 15 minutes to re-run, and get to the next typo :/

Reply:
yeah, it takes much time to rebuild :( Would it help to use the binary instead (`aptos-core/target/debug/aptos-release-builder`)?

Reply:
I will merge this PR for now. Please feel free to submit another when it's ready.